Fastlane/testing_lanes.rb

```diff
@@ -52,6 +54,7 @@
     end
 
     scan(
+      step_name: options[:step_name] || "Scan - #{scheme}",
```

Comment:
This is great!

Reply:
Yeah! I explored other fastlane files in repos like RevenueCat's and found this one, super neat!